ReactWindows/ReactNative.Shared/Views/Text/FontStyleHelpers.cs

```diff
@@ -11,6 +12,20 @@ namespace ReactNative.Views.Text
 {
     static class FontStyleHelpers
     {
+        public static FontStyle? ParseFontStyle(string fontStyleString)
+        {
+#if WINDOWS_UWP
+            return EnumHelpers.ParseNullable<FontStyle>(fontStyleString);
```

Comment:
why have a unique UWP implementation here at all? It seems like it might allow for UWP-specific styles, which would impair component/code sharing between RN platforms for teams who deliver RN apps.

Reply:
You’re right. Ok, I’ll refactor this.